### PR TITLE
Patch 2

### PR DESCRIPTION
**Pull request by dannylewis-sheffield**
Had to import Foundation and CryptoSwift to get 2.1.0 to build, so I've imported them and bumped the version in JSONWebToken.podspec

### CODE REVIEW

File: Sources/ClaimSet.swift [hunk 1]
```diff
@@ -1,3 +1,6 @@
+import Foundation
+import CryptoSwift
```
Comment (kylef) [inline Sources/ClaimSet.swift]: I understand that `Foundation` is required, but I don't think `CryptoSwift` should be. There is no use of CryptoSwift throughout this file.

Comment (dannylewis-sheffield) [inline Sources/ClaimSet.swift]: OK, I've removed it

**Comment by kylef** [comment]
I'm not able to reproduce any build problems on master with Swift 3.0.2. Are you able to reproduce the problem still?

```
$ swift --version
Apple Swift version 3.0.2 (swiftlang-800.0.63 clang-800.0.42.1)
Target: x86_64-apple-macosx10.9
```

```
$ swift test
...
Test Suite 'PayloadTests' passed at 2017-02-17 22:17:06.174.
	 Executed 6 tests, with 0 failures (0 unexpected) in 0.001 (0.001) seconds
Test Suite 'JWTPackageTests.xctest' passed at 2017-02-17 22:17:06.174.
	 Executed 36 tests, with 0 failures (0 unexpected) in 0.017 (0.019) seconds
Test Suite 'All tests' passed at 2017-02-17 22:17:06.174.
	 Executed 36 tests, with 0 failures (0 unexpected) in 0.017 (0.020) seconds
```

**Comment by kylef** [comment]
Solved by b605fe2d53a239946ffa81ab7de32daa71fd4aa4

**Pull request closed without merging**